packages/product/integration-tests/__tests__/module.ts

```diff
@@ -202,6 +203,118 @@ describe("Product module", function () {
     })
   })
 
+  describe("update", function () {
```

Comment:
todo: During the update tests we should try to update existing relations, update existing relations and delete existing relations